### DISCUSSION TITLE
What are some good sound effect isolators?

### DISCUSSION THREAD

**Discussion by Aged-String** [Anjok07/ultimatevocalremovergui#1563] [Q&A]
I'm trying to isolate some sound effects from a video game cutscene (specifically Final Fantasy X's Belleve siege, and mainly the bells before the siege, the ship's sounds, and the gunshots). I heard that UVR5 was good in isolating sound effects, so I wanted to know what model can isolate sound effects from a scene.

**Comment by DCWelch** [comment]
I might recommend giving a go to Kim Vocal 1 or 2 - Instrumental Only (then play with normalizing, reverb, and echo post-processing from there). Haven't tried to do something similar myself though, so I could be wrong. There are certainly many options to get from point A to point B... Good luck!
